gui/default/assets/css/overrides.css

```diff
@@ -304,6 +304,73 @@ ul.three-columns li, ul.two-columns li {
     z-index: 980;
 }
 
+/**
+ * Fancytree tweaks
+ */
+
+table.fancytree-ext-table {
+    width: 100%
+}
+
+span.fancytree-expander,
+span.fancytree-icon {
+    vertical-align: middle !important;
+}
+
+/* Break long names not to push content off the screen. */
+span.fancytree-title {
+    word-break: break-all;
+}
+
+/**
+ * Restore Versions modal tweaks
+ */
+
+/* Contain the file list in a fix-sized box with a scrollbar. By
+   default, display 5.5 items at once (depends on the browser, font, and
+   DPI). The container can be resized by the user at will. */
+#restoreTree-container {
+    height: auto;
+    overflow-y: auto;
+    resize: both;
```

Comment:
What's the use-case to resize horizontally? Looks weird when you expand the area beyond its parent horizontally in the video.

Reply:
I actually simply followed the same code as the Ignore Patterns use, but this does help on small screens when dealing with super-long filenames. Otherwise, just 1 file can take the whole screen.